packages/typespec-ts/test/modularUnit/scenarios/models/models.md

```diff
@@ -178,3 +178,89 @@ export async function read(
   return _readDeserialize(result);
 }
 ```
+
+// We need to skip this case due to tcgc issue: https://github.com/Azure/typespec-azure/issues/3088
+# skip: Should get the effective model name
```

Comment:
skip this case due to tcgc issue: https://github.com/Azure/typespec-azure/issues/3088